wordpress-mobile/WordPress-Android
# Remove Tenor feature configuration

Fixes #14259 

This PR removes the unnecessary tenor feature config (now that it's released to everyone). 

To test:
- Go to media screen
- Add gif from tenor
- Notice you can choose it correctly
- Do the same thing from a new post/image block

PR submission checklist:
